pkg/cluster/volumes.go

```diff
@@ -103,10 +103,10 @@ func (c *Cluster) resizeVolumes(newVolume spec.Volume, resizers []volumes.Volume
 
 	for _, pv := range pvs {
 		volumeSize := quantityToGigabyte(pv.Spec.Capacity[v1.ResourceStorage])
-		if volumeSize > newSize {
-			return fmt.Errorf("cannot shrink persistent volume")
-		}
-		if volumeSize == newSize {
+		if volumeSize >= newSize {
+			if volumeSize > newSize {
+				c.logger.Warningf("cannot shrink persistent volume")
```

Comment:
Do we log somewhere in the neighborhood of this statement which exactly volumes/sizes are we talking about ?

Reply:
Hm.. this change seems to be unrelated to the actual PR. Not sure how did it get there.